data/avgi/avgi weapons.txt

```diff
@@ -0,0 +1,664 @@
+	sprite "effect/pwaveflare"
+		"frame rate" 60
+		"random start frame"
+	"lifetime" 1
+	"velocity scale" 1
+
+
+# Missiles:
+
+
+outfit "External Launch Rail"
+	category "Secondary Weapons"
+	licenses
+		"Avgi Atomics"
+	cost 80000
+	thumbnail "outfit/unknown"
+	"mass" 6
+	"outfit space" -6
+	"weapon capacity" -9
+	"gun ports" -1
+	"ophrys capacity" 1
+	weapon
+		sprite "projectile/emp torpedo"
+			"frame rate" 3
+		sound "torpedo"
+		ammo `"Ophrys" Nuclear Torpedo`
+		"stream"
+		icon "icon/nuke"
+		"fire effect" "torpedo fire"
+		"firing energy" 30
+		"firing heat" 15
+		"firing force" 600
+		"reload" 360
+		"velocity" 6
+		"lifetime" 3
+		"turn" 90
+		"homing" 2
+		"optical tracking" 0.72
+		"range override" 600
+		"stream"
+		"submunition" "Ophrys Active"
+	description "This pylon externally mounts a single torpedo, employing a small electromagnetic catapult to fling is payload towards a nearby target."
+
+
+outfit "External Launch Tube"
+	category "Secondary Weapons"
+	licenses
+		"Twilight Guard"
+	cost 1370000
+	thumbnail "outfit/unknown"
+	"mass" 6
+	"outfit space" -24
+	"weapon capacity" -24
+	"gun ports" -1
+	"orchid capacity" 6
+	weapon
+		sprite "projectile/shardactive"
+			"no repeat"
+			"frame rate" .25
+		sound "finisher"
+		ammo `"Orchid" Nuclear Missile`
+		icon "icon/finisher"
+		"fire effect" "sidewinder fire"
+		"fire effect" "speckle fire"
+		"inaccuracy" 1.2
+		"velocity" 6
+		"lifetime" 30
+		"reload" 180
+		"firing energy" 60
+		"firing heat" 30
+		"drag" 3
+		"radar tracking" 0.9
+		"homing" 4
+		"missile strength" 9
+		"stream"
+		"submunition" "Orchid Active"
+	description "This is a large launcher capable of carrying and deploying large torch-driven missiles and torpedoes. With a standardized design, it is compatible with a wide range of Avgi torch missiles. A coilgun-based ejection system allows missile to be deployed with considerable initial velocity."
+
+
+outfit "Vertical Launch Array"
+	category "Secondary Weapons"
+	licenses
+		"Twilight Guard"
+	cost 1370000
+	thumbnail "outfit/unknown"
+	"mass" 9
+	"outfit space" -45
+	"weapon capacity" -45
+	"turret mounts" -1
+	"nettle capacity" 240
+	weapon
+		sprite "projectile/shardactive"
+			"no repeat"
+			"frame rate" .25
+		sound "finisher"
+		ammo `"Nettle" KKV`
+		icon "icon/finisher"
+		"fire effect" "sidewinder fire"
+		"inaccuracy" 360
+		"velocity" 3
+		"lifetime" 30
+		"reload" 15
+		"burst reload" 9
+		"burst count" 12
+		"turret turn" 9
+		"firing energy" 3
+		"firing heat" 1.5
+		"infrared tracking" 0.9
+		"optical tracking" 0.6
+		"homing" 4
+		"missile strength" 9
+		"submunition" "Nettle Active"
+	description "This large Vertical Launch System is made up independent launch cells arrayed in parallel. A wide variety of small to mid-sized Avgi missiles are compatible with the launch cells, which can be easily interchanged. Designed for rapid vertical cold-launch, the system needs an open space comparable to a turret mount to function properly."
```

Comment:
"made up of". can't be assed to remember how to make it a fancy edit thingy sorry